### ISSUE TITLE
app history version

### ISSUE THREAD

**Issue by gitxinyi** [dyseo/NewApkPure#2]
Have you studied the download history of the app

**Comment by Alnyz** [comment]
> Have you studied the download history of the app

Pardon, what you mean download history ?

**Comment by gitxinyi** [comment]
Sorry, I didn't get the description. apk resources of different historical versions are downloaded

**Comment by Alnyz** [comment]
> Sorry, I didn't get the description. apk resources of different historical versions are downloaded

so you, want download download with specific version?

**Comment by gitxinyi** [comment]
Yes, the latest version of the website has access to the api, the historical version cannot be downloaded

**Comment by Alnyz** [comment]
> Yes, the latest version of the website has access to the api, the historical version cannot be downloaded

i see, i will check for the newest update of website and update the library ASAP.

thanks for information
